dcc-portal-ui/app/scripts/survivalanalysis/js/survival-analysis.js

```diff
@@ -309,6 +337,44 @@
         update();
       };
 
+      this.handleClickExportSvg = () => ExportService.exportData('survivalplot.svg', `
+        <svg
+          xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cc="http://creativecommons.org/ns#"
+          xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
```

Comment:
monkey-see monkey-do, not quite familiar with xmlnss
read [an article](https://developer.mozilla.org/en/docs/Web/SVG/Namespaces_Crash_Course), still not sure if the extra ones are required
will remove, should still work